app/views/shared/_sub_nav.html.erb

```diff
@@ -8,7 +8,7 @@
       </button>
       <div class="collapse navbar-collapse" id="subnavbarNavDropdown">
         <ul class="navbar-nav">
-          <%= render 'shared/dashboards' if current_user %>
+          <%= render 'shared/dashboards' if can?(:read, :dashboard_menu) && school.active? %>
```

Comment:
and here too, but this school might not be the current user's school

Reply:
I don't think it matters if it's their school or not? We just need to check they're the right role and the school is active?